craft_parts/callbacks.py

```diff
@@ -33,7 +33,10 @@
 StepCallback = Callable[[StepInfo], bool]
 ConfigureOverlayCallback = Callable[[Path, ProjectInfo], None]
 Callback = Union[
-    FilterCallback, ExecutionCallback, StepCallback, ConfigureOverlayCallback
+    FilterCallback,
```

Comment:
did we change the desired line length?

Reply:
No, but the `ruff format` seems like break long lines.